content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -32,35 +32,23 @@ When enabling non-interactive access to RKVST, you ***must*** create your first
 
 ### Using the RKVST UI to create an App Registration (First-Time Setup)
 
-1. As an Administrator, open the `Settings` interface.
-
-{{< img src="Settings.png" alt="Rectangle" caption="<em>Settings</em>" class="border-0" >}}
-
-2. Navigate to the `APP REGISTRATIONS` tab.
-
-{{< img src="AppRegistrationsTab.png" alt="Rectangle" caption="<em>Navigate to APP REGISTRATIONS</em>" class="border-0" >}}
-
-3. Click `CREATE APP REGISTRATION` and the following form should appear:
-
-{{< img src="AppRegistrationForm.png" alt="Rectangle" caption="<em>App Registration Webform</em>" class="border-0" >}}
-
-3. Enter any display name you'd like, then click `CREATE APP REGISTRATION`.
-
-{{< note >}}
- You can optionally add any Custom Claims at this step. You must ensure they do not start with `jit_` or use any of the [well-known reserved claims](https://auth0.com/docs/security/tokens/json-web-tokens/json-web-token-claims#reserved-claims). The Custom Claims can be used in an [Attribute-Based Access Control (ABAC) policy](/platform/administration/managing-access-to-an-asset-with-abac) to grant permissions. 
-{{< /note >}}
-
-{{< img src="CreateAppRegistration.png" alt="Rectangle" caption="<em>Completed Web Registration</em>" class="border-0" >}}
-
-4.  You will then be presented with the `CLIENT_ID` and `SECRET` required by the archivist token endpoint.
-
-{{< caution >}}
-**Caution:** You **must** take note of the `SECRET` at this point - it can **not** be viewed again later and you will have to generate a new one.
-{{< /caution >}}
-
-{{< img src="RecordClientIDandSecret.png" alt="Rectangle" caption="<em>Record your Client ID and Secret</em>" class="border-0" >}}
-
-5. Now that you have created your App Registration, follow the steps below to [test generating a token](./#getting-a-token-with-your-app-registration) and [ensure you can access the RKVST API](./#testing-your-access).
+1. As an Administrator, open the [RKVST Portal](https://app.rkvst.io/)
```

Comment:
Called out the URL so users can click directly.  I haven't figured out how to get HUGO to open a link in a new tab, similar to [just the docs](https://github.com/just-the-docs/just-the-docs) does with a [{:target="_blank"}](https://github.com/scitt-community/scitt-web/blob/main/scitt-specs.md?plain=1#L13) addition

Reply:
I like this, but we should still be able to tell the story if we don't have screenshots. They are an overhead in the docs and we are aiming to reduce the use of them, where we can.